hibike/hibike_message.py

```diff
@@ -89,11 +91,11 @@ def getDeviceType(uid):
 # return bits [71: 64] of the UID
 def getYear(uid):
   temp = uid >> 64
-  return int(temp & 0xff)
+  return int(temp & 0xFF)
 
 # return bits[63: 0] of the UID
 def getID(uid):
-  return uid & 0xffffffffffffffff
+  return uid & 0xFFFFFFFFFFFFFFFF
```

Comment:
It's a 64 bit bit-mask. Would replacing this with a variable actually make it more readable?